eng/WpfArcadeSdk/tools/Pbt.targets

```diff
@@ -101,4 +101,23 @@
   <Import Sdk="Microsoft.NET.Sdk.WindowsDesktop"
           Project="../targets/Microsoft.NET.Sdk.WindowsDesktop.targets"
           Condition="'$(InternalMarkupCompilation)'=='true' And !Exists('$(LocalMicrosoftWinFXTargets)') "/>
+
+  <!--
+  WindowsBase comes from NetCoreApp.Reference. When we try to compile the themes dlls, which reference WindowsBase,
```

Comment:
Maybe I am late on this, but do we have a binlog from a prior build that proves they were getting the wrong WindowsBase in wpf-int?  If the answer is no, then I don't think we should make this change, but fix the underlying reason why this has changed in the port to GitHub.

Reply:
Ahh ok. i just looked at one of the binlogs in wpf-int and it looks like `WindowsBase` gets referenced from the `Microsoft.DotNet.Wpf.GitHub` package:

> Encountered conflict between 'Platform:WindowsBase.dll' and 'Reference:F:\workspace\_work\1\s\.packages\runtime.win-x64.microsoft.dotnet.wpf.github\4.8.0-preview9.19408.5\lib\netcoreapp3.0\WindowsBase.dll'. Choosing 'Reference:F:\workspace\_work\1\s\.packages\runtime.win-x64.microsoft.dotnet.wpf.github\4.8.0-preview9.19408.5\lib\netcoreapp3.0\WindowsBase.dll' because file version '4.800.19.40805' is greater than '4.700.19.40503'.

So WindowsBase from GitHub Nuget was being chosen over WindowsBase from NetCore.App due to the file version being greater. Now that we are referencing WindowsBase as a ProjectReference, this doesn't happen and so we now get build breaks because there already is a reference to WindowsBase.

At this point we should be able to delete/refactor a lot of the WpfArcadeSdk now that all the managed code is on GitHub.  However,I don't believe this is the time or place for that. I'd prefer a small, simple, and targeted fix for right now that unblocks this PR rather than trying to engineer something more complicated. I don't think fix is technically incorrect either,

Reply:
Ok that makes some more sense I think.  But shouldn't the version from the project reference beat NetCore.App?  This could be a quirk with RAR that we have to have another workaround on top for.

Is this happening elsewhere?  Like won't everyone using WindowsBase as a project reference have this issue?  Why does just PBT encounter this?

Reply:
There is a known issue around ProjectReference not being considered, see https://github.com/dotnet/sdk/issues/2674

Since there are two WindowsBase references, PBT is trying to load both into the same MetadataLoadContext, and so an exception is thrown since this isn't allowed. Our test projects don't actually ProjectReference WindowsBase, they use a regular Reference instead*. 

*They actually do the same thing as I've done here with removing WindowsBase from NetCore.App but it isn't necessary since referencing the DLL will overwrite the one that comes from NetCore.App.

Reply:
I believe the reason we are hitting it now is because the themes dlls explicitly reference WindowsBase in the markup like this:

```
xmlns:base="clr-namespace:System.Windows;assembly=WindowsBase"
```

which isn't a common practice. do you know why they do that?

Reply:
ProjectReference not being factored in conflict-resolution during ResolveAssemblyReferences is indeed a problem. The solution for that in our builds is to side-step the problem by keeping NetCore.app.Ref copy of Windowsbase out of the reference-list. This is done by way of using NetCoreReference.

Reply:
Just to clarify, we reference WindowsBase for flags.  The themes have to know about various compat flags for accessibility (or they did in .NET Framework).  The best place to generate those was in Base because we set all that up in the Dispatcher's static constructor.

That's probably not needed right now, but that's a large change to generated code that we don't really want to make without the appropriate generators enabled.  So, for now, this sort of has to stick around.

Reply:
> I believe the reason we are hitting it now is because the themes dlls explicitly reference WindowsBase in the markup like this:

This might be the first-order cause in a manner of speaking, but I'm doubtful that this is the problem. Please follow up on my [suggestion ](https://github.com/dotnet/wpf/pull/1576#discussion_r312287808) and report back - we'll figure out next steps if that doesn't work.

Reply:
Let me correct myself, those can't be removed regardless, they're needed for 4.8 compat.  5.0 is the first candidate for removing them as they'll break compat expectations with .NET 4.8.